test/e2e/components/vl-alert.js

```diff
@@ -41,7 +41,10 @@ class VlAlert extends VlElement {
     }
 
     async getActions() {
-        return this._innerText('#actions');
+      const slot = await (await this._getAlert()).findElement(
+          By.css("#actions-slot"));
+      return this.driver.executeScript(
+          'return arguments[0].assignedElements()', slot);
```

Comment:
moeten we dit toch eens proberen in de base class te steken? precies al 2e keer dat ik dit zie passeren in verschillende reviews?

Reply:
Nog niet opgelost hier, maar wel al in de base class gestoken